src/main/java/duke/data/task/Deadline.java

```diff
@@ -0,0 +1,35 @@
+package duke.data.task;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.time.temporal.ChronoUnit;
+public class Deadline extends Task {
+    private String deadline;
+
+    /**
+     * Constructor to initialize Deadline.
+     *
+     * @param description Description of the deadline task.
+     * @param deadline Date of deadline.
+     */
+    public Deadline(String description, String deadline) {
+        super(description);
+        this.deadline = deadline;
+        try {
+            LocalDate d1 = LocalDate.parse(deadline);
```

Comment:
Instead of d1, maybe can use dueDate? (use name to explain)
```suggestion
            LocalDate dueDate = LocalDate.parse(deadline);
```